google-cloud-kms/acceptance/google/cloud/kms/v1/smoke_test.rb

```diff
@@ -17,12 +17,12 @@
 
 require "google/cloud/kms"
 
-class KmsServiceSmokeTest < Minitest::Test
+class KmsServiceSmokeTest < Minitest::Spec
   def test_list_key_rings
     kms = Google::Cloud::Kms.key_management_service version: :v1
-    key_ring_parent = kms.class.location_path ENV["KMS_PROJECT"], "us-central1"
+    key_ring_parent = kms.location_path project: ENV["KMS_PROJECT"], location: "us-central1"
     key_rings = kms.list_key_rings(parent: key_ring_parent).to_a
-    key_rings.size.must_equal 1
-    key_rings[0].name.must_match %r{keyRings/ruby-test$}
+	_(key_rings.size).must_equal 1
```

Comment:
One other thing, can you fix the indentation here?

Reply:
Oh, I think it happened because I was using tab instead of space for indentaion.